README.md

```diff
@@ -117,7 +116,7 @@ $ rails new . -d postgresql
 ```
 
 ```
-Your answer...
+The first command creates a rails app in a new folder called tunr.  The second command creates the app in the current folder.  The second one also includes a link to a postgresql database.
```

Comment:
:+1:

Instead of saying it includes a "link to a postgresql database", I recommend indicating that it configures the new app to use a postgresql database.